api/src/Entity/User.php

```diff
@@ -233,6 +236,8 @@ public function getRoles(): array
         // guarantee every user at least has ROLE_USER
         $roles[] = 'ROLE_USER';
 
+        return $this->roles;
+
         return array_unique($roles);
     }
```

Comment:
```suggestion
        return $this->roles;

        return array_unique($roles);
    }
```
Warning you have 2 return. You don't keep `return array_values(array_unique($roles));`?